### DISCUSSION TITLE
Authenticator pin

### DISCUSSION THREAD

**Discussion by elderkenshi** [akopachov/flipper-zero_authenticator#101] [Q&A]
How can I reset the authenticator pin? I forgot it!

**Comment by akopachov** [comment]
There is no way to reset\change pin once it is already set and you forgot it without loosing all the tokens. Otherwise such a pin won't make any sense.

However you can completely reset application including PIN, but keep in mind that you will loose all the tokens and settings you created

[How to change PIN](https://github.com/akopachov/flipper-zero_authenticator/wiki/How-to-change%5Crecover-PIN%3F)

[How to reset application](https://github.com/akopachov/flipper-zero_authenticator/wiki/How-to-reset-application%3F)

**Comment by elderkenshi** [comment]
Just to be clear, I mean the authenticator located in: Applications/Misc/Authenticator

**Reply by akopachov** [reply]
okay, that is right Authenticator and it doesn't change the answer [given above](https://github.com/akopachov/flipper-zero_authenticator/discussions/101#discussioncomment-5481164).

**Comment by elderkenshi** [comment]
I´m not a developer just a tech-enthusiast, do you believe is it possible to write some sort of script that runs over the qFlipper, and triggers all the possible arrow combinations that the password might have? If I know the exact amount of arrows, would this script be able to find the combination?

**Reply by akopachov** [reply]
yes, I pretty sure it is doable.
